src/ripple/protocol/STTx.h

```diff
@@ -49,7 +49,7 @@ class STTx final : public STObject, public CountedObject<STTx>
     }
 
     static std::size_t const minMultiSigners = 1;
-    static std::size_t const maxMultiSigners = 8;
+    static std::size_t const maxMultiSigners = 100;
```

Comment:
This is a problem: this change needs to be gated under an amendment; if the amendment is not active, the old value is used. The new only value only kicks in if the amendment is active.

Take a look at https://github.com/ripple/rippled/pull/3580 which makes some changes to the way signer lists are handled. It will show how to properly gate your code.